tests/test_terraform.py

```diff
@@ -29,33 +29,62 @@
 class TestDeployVPC(unittest.TestCase):
     def test_happy_path(self):
         executor = TerraformExecutor(
-            terraform_workspace='test',
-            terraform_vars_file='terraform_single_az.tfvars',
+            terraform_workspace="test",
+            terraform_vars_file="terraform_single_az.tfvars",
             terraform_vars={"execution_id": str(uuid.uuid4())},
-            module_path='tests/vpc-terraform',
+            module_path="tests/vpc-terraform",
             log_level=TerraformLogLevel.INFO,
         )
         try:
             results = executor.deploy()
-            self.assertEqual(0, results.returncode, msg=f'Deployment was not successful, check the session output')
+            self.assertEqual(
+                0,
+                results.returncode,
+                msg=f"Deployment was not successful, check the session output",
+            )
             outputs = executor.output()
-            self.assertIsNotNone(outputs['vpc_id'])
-            self.assertEqual(1, len(outputs['private_subnet_ids']))
-            self.assertEqual(0, len(outputs['public_subnet_ids']))
+            self.assertIsNotNone(outputs["vpc_id"])
+            self.assertEqual(1, len(outputs["private_subnet_ids"]))
+            self.assertEqual(0, len(outputs["public_subnet_ids"]))
         finally:
             executor.destroy()
 
+
 class TestDeployClusterUsingCloudQ(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.vpc_executor = TerraformExecutor(
+            terraform_workspace="test",
+            terraform_vars_file="terraform_multi_az.tfvars",
```

Comment:
We're doing multi-AZ for the entire test class?

I'd like to understand your thoughts around this

Reply:
Previously, the test passed with multi AZ so we are trying out using the generated vpc and subnet to see if the test will still pass. And then we can remove those manual created vpc and subnets. Trying to add 1 thing at a time. However, the terraform is running nonstop after we moved to generated vpc and subnet. I am still trying to figure out what is wrong.
```
module.qprovisioner.null_resource.provisioner_status: Still creating... [57m30s elapsed]
module.qprovisioner.null_resource.provisioner_status (local-exec): Waiting for boot...
```